bungee/src/main/java/net/minevn/crosswhisper/bungee/CWBungee.java

```diff
@@ -71,6 +74,22 @@ public void sendMessage(ProxiedPlayer sender, ProxiedPlayer receiver, String mes
                 .replace("%receiver%", receiver.getName())
                 .replace("%message%", message)
         );
+
+        for(ProxiedPlayer spy : spyMap.keySet()) {
```

Comment:
1. `spyMap` có khả năng được thao tác bởi nhiều threads (vì Bungeecord chạy đa luồng), vì vậy `spyMap` nên dùng `ConcurrentHashMap` 
2. Chỗ iterate map `spyMap` nên viết là `for (ProxiedPlayer entry : spyMap.entrySet())`, rồi sau đó `entry.getKey()` và `entry.getValue()` bên trong `for`